xbmc/input/actions/ActionIDs.h

```diff
@@ -32,132 +32,192 @@
+
+//! Zoom 5x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_4 41
+
+//! Zoom 6x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_5 42
+
+//! Zoom 7x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_6 43
+
+//! Zoom 8x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_7 44
+
+//! Zoom 9x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_8 45
+
+//< Zoom 10x picture during slideshow. Can be used in slideshow.xml window id=2007
+#define ACTION_ZOOM_LEVEL_9 46
+
+//< Select next arrow. Can be used in: settingsScreenCalibration.xml windowid=11
+#define ACTION_CALIBRATE_SWAP_ARROWS 47
+
+//! Reset calibration to defaults. Can be used in: `settingsScreenCalibration.xml`
+//! windowid=11/settingsUICalibration.xml windowid=10
+#define ACTION_CALIBRATE_RESET 48
+
+//! Analog thumbstick move. Can be used in: `slideshow.xml`
+//! windowid=2007/settingsScreenCalibration.xml windowid=11/settingsUICalibration.xml
+//! windowid=10
+//! @note see also ACTION_ANALOG_MOVE_X_LEFT, ACTION_ANALOG_MOVE_X_RIGHT,
+//! ACTION_ANALOG_MOVE_Y_UP, ACTION_ANALOG_MOVE_Y_DOWN
+#define ACTION_ANALOG_MOVE 49
+
+//! Rotate current picture clockwise during slideshow. Can be used in
+//! slideshow.xml window < id=2007
+#define ACTION_ROTATE_PICTURE_CW 50
+
+//! Rotate current picture counterclockwise during slideshow. Can be used in
+//! slideshow.xml window id=2007
+#define ACTION_ROTATE_PICTURE_CCW 51
+
+//! Decrease subtitle/movie Delay.  Can be used in videoFullScreen.xml window id=2005
+#define ACTION_SUBTITLE_DELAY_MIN 52
+
+//! Increase subtitle/movie Delay.  Can be used in videoFullScreen.xml window id=2005
+#define ACTION_SUBTITLE_DELAY_PLUS 53
+
+//! Increase avsync delay.  Can be used in videoFullScreen.xml window id=2005
+#define ACTION_AUDIO_DELAY_MIN 54
+
+//! Decrease avsync delay.  Can be used in videoFullScreen.xml window id=2005
+#define ACTION_AUDIO_DELAY_PLUS 55
+
+//! Select next language in movie.  Can be used in videoFullScreen.xml window id=2005
+#define ACTION_AUDIO_NEXT_LANGUAGE 56
+
+//! Switch 2 next resolution. Can b used during screen calibration
+//! settingsScreenCalibration.xml windowid=11
+#define ACTION_CHANGE_RESOLUTION 57
+
+#define REMOTE_0 58 //!< Remote keys 0-9. are used by multiple windows
+#define REMOTE_1 59 //!< For example in videoFullScreen.xml window id=2005 you can
+#define REMOTE_2 60 //!< Enter time (mmss) to jump to particular point in the movie
 #define REMOTE_3 61
-#define REMOTE_4 62 //!< with spincontrols you can enter 3digit number to quickly set
+#define REMOTE_4 62 //!< With spincontrols you can enter 3digit number to quickly set
 #define REMOTE_5 63 //!< spincontrol to desired value
 #define REMOTE_6 64
 #define REMOTE_7 65
 #define REMOTE_8 66
 #define REMOTE_9 67
```

Comment:
Think to use here as:
```cpp
//! Remote keys 0-9. are used by multiple windows.
//!
//! For example in videoFullScreen.xml window id=2005 you can
//! Enter time (mmss) to jump to particular point in the movie
//!
//! With spincontrols you can enter 3digit number to quickly set
//! spincontrol to desired value.
//!@{
#define REMOTE_0 58

//! @see REMOTE_0 about details.
#define REMOTE_1 59

//! @see REMOTE_0 about details.
#define REMOTE_2 60

//! @see REMOTE_0 about details.
#define REMOTE_3 61

//! @see REMOTE_0 about details.
#define REMOTE_4 62

//! @see REMOTE_0 about details.
#define REMOTE_5 63

//! @see REMOTE_0 about details.
#define REMOTE_6 64

//! @see REMOTE_0 about details.
#define REMOTE_7 65

//! @see REMOTE_0 about details.
#define REMOTE_8 66

//! @see REMOTE_0 about details.
#define REMOTE_9 67
//!@}
```

With a `//!<` becomes everything added independently to his define.

Here how looks:
Before:
![Bildschirmfoto vom 2020-10-29 20-01-37](https://user-images.githubusercontent.com/6879739/97621344-18f7c280-1a23-11eb-946a-28776fd5b28e.png)

---------------------------------------------

After:
![Bildschirmfoto vom 2020-10-29 20-13-24](https://user-images.githubusercontent.com/6879739/97621579-5e1bf480-1a23-11eb-862b-1c78a0ed6afe.png)